waku/v2/protocol/waku_store/waku_store_types.nim

```diff
@@ -35,6 +35,8 @@ const
 
   MaxRpcSize* = MaxPageSize * MaxWakuMessageSize + 64*1024 # We add a 64kB safety buffer for protocol overhead
 
+  MaxTimeVariance* = (20*1000*1000*1000).int64 # 20 seconds maximum allowable sender timestamp "drift" into the future
```

Comment:
Same as above, you can use [getNanosecondTime](https://github.com/status-im/nim-waku/blob/8ed31ffddb6b755ad75e7d48f6669454f26ed01a/waku/v2/utils/time.nim#L11) to avoid such multiplier.

Reply:
Wondering why not using the geTNanosecondTime() proc? using the same proc lowers the chance of unwanted errors